src/workflow/StreamlitUI.py

```diff
@@ -42,17 +44,20 @@ def upload_widget(
 
         Args:
             key (str): A unique identifier for the upload component.
-            file_type (str): Expected file type for the uploaded files.
+            file_types (Union[str, List[str]]): Expected file type(s) for the uploaded files.
             name (str, optional): Display name for the upload component. Defaults to the key if not provided.
             fallback (Union[List, str], optional): Default files to use if no files are uploaded.
+            symlink (bool, optional): Whether to symlink the uploaded files. This is used if local to avoid making duplicate copies. Defaults to True.
```

Comment:
```suggestion
            symlink (bool, optional): Whether to symlink the uploaded files. This is used if local to avoid making duplicate copies. Defaults to False.
```